src/aims_dev_ui/routes.py

```diff
@@ -100,7 +100,7 @@ def endpoints(endpoint_path):
         results = error
     else:
       try:
-        response = requests.get(uri, params=form.data)
+        response = requests.get(uri, params=form.data, headers={'Authorization': 'Bearer ' + app.config['SECRET_KEY']})

```

Comment:
and again....

Reply:
I have created a separate ticket for the exeter api which (wrongly) has Basic Auth. To apply this now would be jumping the gun.

Reply:
This can be done in a new PR as it's not required yet. Nice and simple to implement.